### PR TITLE
Added Context Menu Entries For Wayback Machine

### PR DESCRIPTION
**Pull request by collinx**
This PR is towards adding the Context Menu entries for Wayback Machine Extension in Chrome Browser. The user can right-click on a web page to access the child entries save page now, recent version and first version under Wayback Machine Entry.

contextmenu

@rchrd2 @markjohngraham Kindly review this PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,9 +37,9 @@
     <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
   </body>
 </html>
-<script src="scripts/jquery.js"></script>
-<script src="scripts/popup.js"></script>
-<script src="scripts/bootstrap.min.js"></script>
+
+<script type="text/javascript" src="scripts/popup.js"></script>
+<script type="text/javascript" src="scripts/bootstrap.min.js"></script>
```
Comment (collinx) [inline index.html]: @rgpt  thanks for taking the time to go through my code and reviewing it. As for your review, I have already seen the error and it is a custom exception which is thrown if jquery is not there. Even if it is being shown on the console, it doesn't have any effect on speed or efficiency of extension.

**Comment by collinx** [comment]
@rchrd2 @markjgraham Kindly review this PR. New changes to the v1.3.3 branch have been updated here.

**Comment by collinx** [comment]
@rchrd2 @markjgraham Kindly review this PR. New changes to the v1.3.3 branch have been updated here.

**Comment by collinx** [comment]
@rakesh-chinta I have removed the conflicts. Have a look at this PR.

**Comment by rakesh-chinta** [comment]
Thank you for the update @collinx, but I think you may have missed one more conflicting file,
i.e " scripts/background.js", 
let me know when it is fixed, good work!

**Comment by collinx** [comment]
@rakesh-chinta rectified the conflicts.

**Pull request merged**